packages/rum-core/src/domain/rumEventsCollection/view/trackInitialViewTimings.ts

```diff
@@ -121,15 +124,16 @@ export function trackLargestContentfulPaintTiming(
   const { unsubscribe: unsubscribeLifeCycle } = lifeCycle.subscribe(
     LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED,
     (entries) => {
-      for (const entry of entries) {
```

Comment:
Ah you shouldn't use `find` for LCP as there may be more than one, and the last one should be used, not the first one. Maybe implement a `findLast` helper?

It's for the `first-input` that you could use a `find`